queue_services/entity-filer/src/entity_filer/filing_processors/incorporation_filing.py

```diff
@@ -114,9 +125,69 @@ def _update_cooperative(incorp_filing: Dict, business: Business, filing: Filing)
+        certified_rules_file_name = CERTIFIED_COPY_PREFIX + rules_file_name
+        certified_rules_file_key = _create_certified_copy(rules_file.data,
+                                                         business,
+                                                         certified_rules_file_name)
+
+        document = Document()
+        document.type = DocumentType.CERTIFIED_COOP_RULES.value
+        document.file_key = certified_rules_file_key
+        document.file_name = certified_rules_file_name
+        document.content_type = document.file_name.split('.')[-1]
+        document.business_id = business.id
+        document.filing_id = filing.id
+        business.documents.append(document)
+
+        # create certified copy for memorandum document
+        memorandum_file = MinioService.get_file(cooperative_obj.get('memorandumFileKey'))
+        memorandum_file_name = cooperative_obj.get('memorandumFileName')
+        certified_memorandum_file_name = CERTIFIED_COPY_PREFIX + memorandum_file_name
+        certified_memorandum_file_key = _create_certified_copy(memorandum_file.data,
+                                                              business,
+                                                              certified_memorandum_file_name)
+
+        document = Document()
+        document.type = DocumentType.CERTIFIED_COOP_MEMORANDUM.value
+        document.file_key = certified_memorandum_file_key
+        document.file_name = certified_memorandum_file_name
+        document.content_type = document.file_name.split('.')[-1]
+        document.business_id = business.id
+        document.filing_id = filing.id
+        business.documents.append(document)
+
     return business
 
 
+def _create_certified_copy(_bytes, business, certified_file_name):
+    open_pdf_file = io.BytesIO(_bytes)
+    pdf_reader = PyPDF2.PdfFileReader(open_pdf_file)
+    pdf_writer = PyPDF2.PdfFileWriter()
+    pdf_writer.appendPagesFromReader(pdf_reader)
+    output_original_pdf = io.BytesIO()
+    pdf_writer.write(output_original_pdf)
+    output_original_pdf.seek(0)
+
+    registrar_info = RegistrarInfo.get_registrar_info(business.founding_date)
+    registrars_signature = registrar_info['signatureAndText']
+    pdf_service = PdfService()
+    registrars_stamp = pdf_service.create_registrars_stamp(registrars_signature,
+                                                           LegislationDatetime.as_legislation_timezone(business.founding_date),
+                                                           business.identifier)
+    certified_copy = pdf_service.stamp_pdf(output_original_pdf, registrars_stamp, only_first_page=True)
+
+    signed_url = MinioService.create_signed_put_url(certified_file_name, prefix_key=COOPERATIVE_FOLDER_NAME)
+    key = signed_url.get('key')
+    pre_signed_put = signed_url.get('preSignedUrl')
+
+    requests.put(pre_signed_put, data=certified_copy.read(), headers={'Content-Type': 'application/octet-stream'})
```

Comment:
I am not sure why we create a new file instead of replacing the existing one?

Reply:
yes, discuss it but as we don't decide it, let me confirm it.

Reply:
yeah i think we decided we would replace the existing file.

Reply:
OK. Now it is creating a stamped file and remove the original in the post_process.

Reply:
Now that the file key in filing_json and the document table are different. I don't see any issue with that, just to be clear @thorwolpert can you confirm

Reply:
I couldn't find a way to overwrite the original one with minio. Another issue is: if the there is an error on the second one, then we would have one stamped and the second without stamp in minio. I'm not sure if that would be a major problem.